### ISSUE TITLE
java process is not killed after quiting nvim

### ISSUE THREAD

**Issue by vcfvct** [neoclide/coc-java#49]
Seems the `java` process does not exit after vim quit. Is there any `lifecycle hooks` to ensure the resource getting released? 

`/Library/Java/JavaVirtualMachines/jdk1.8.0_181.jdk/Contents/Home/bin/java -XX:MaxPermSize=256m -XX:+HeapDumpOnOutOfMemoryError -Xmx1024m -Dfile.encoding=utf-8 -Duser.country=US -Duser.language=en -Duser.variant -cp /Users/myUser/.gradle/wrapper/dists/gradle-3.5-bin/a9dfgtj5kwo69rnsu3ha320bn/gradle-3.5/lib/gradle-launcher-3.5.jar org.gradle.launcher.daemon.bootstrap.GradleDaemon 3.5`

System: MacOS Mojave
nvim: v0.4.2
![Screen Shot 2019-10-16 at 1 12 27 PM](https://user-images.githubusercontent.com/2096033/66942246-a6ce5e80-f016-11e9-986a-7c083b4e2126.png)

Thanks for the awesome `coc.vim`!

**Comment by patrickpichler** [comment]
Gradle is starting a background daemon for faster compilation times. So if you execute lets say `gradle build` gradle will start a daemon as well. So this is not really related to `coc-java`. 

Heres a link to the official gradle documentation https://docs.gradle.org/current/userguide/gradle_daemon.html

**Comment by vcfvct** [comment]
@patrickpichler do you mean the `jdt` language server is starting this? FYI I did not run any `gradle` command, just `nvim xxxx.java` and that process will be brought up.

**Comment by patrickpichler** [comment]
Exactly. `jdt` uses the eclipse gradle stuff (if I'm not mistaken) and this starts the gradle daemon to get informations about the build.

**Comment by chemzqm** [comment]
Should be fixed on recent version of coc-java, try upgrade coc.nvim to latest release and coc-java to version 1.14.0.
